app/controllers/account/companies_controller.rb

```diff
@@ -7,6 +7,7 @@ def index
 
   def show
     @company = current_company
+    @videos = @company.videos.first(3)
```

Comment:
use `#take` instead of `#first`
http://www.rubydoc.info/github/rails/rails/master/ActiveRecord/FinderMethods:take